components/cosmetic_filters/resources/data/content_cosmetic.ts

```diff
@@ -176,13 +176,17 @@ const UseMutationObserver = () => {
 }
 
 const UseSelectorsPolling = () => {
+  const mutations = cosmeticObserver?.takeRecords()
```

Comment:
whats the expected condition where `cosmeticObserver` is `undefined` here?  Why the `?` guard? (just trying to reduce number of cases where we're swallowing errors to make it easier to figure out where the un-caught cases are coming from